alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/transport/TransportIndexDestinationAction.kt

```diff
@@ -88,6 +91,19 @@ class TransportIndexDestinationAction @Inject constructor(
         }
 
         private fun prepareDestinationIndexing() {
+
+            // Prevent indexing if the Destination type is disallowed
+            val destinationType = request.destination.type.value
+            if (!allowList.contains(destinationType)) {
+                actionListener.onFailure(
+                    ElasticsearchStatusException(
```

Comment:
Why is there no `AlertingException.wrap()`?

Reply:
Added.